modules/waf/main.tf

```diff
@@ -0,0 +1,9 @@
+##############################################################################
+# Enable Web Application Firewall for a domain
```

Comment:
It will be used for enabling and disabling both, so write accordingly.

Reply:
Made the changes.